### DISCUSSION TITLE
debian packaging

### DISCUSSION THREAD

**Discussion by hsbasu** [BuddiesOfBudgie/budgie-control-center#88] [General]
Hi! Budgie team,
    I know budgie has debian packaging downstream by Ubuntu Budgie team. But do you have plans to keep debian packaging on github for testing purposes?

**Comment by fossfreedom** [comment]
not really sure what you are asking here.

Correct - the ubuntu budgie repo is specifically for the maintenance of the deb package in Debian itself.

Any variants of the deb package is specific to a purpose - publicly available PPAs on launchpad/~ubuntubudgie-dev - for example the daily PPA.

There isn't any maintenance here upstream for deb packages - same as for other distros.

**Comment by carlosbrad** [comment]
You could add support again for online accounts by placing the gnome-online-accounts-gtk package as a dependency, which works in any Linux desktop environment without the need to use gnome, just put a link to it in budgie-control- center to use it for online accounts
